molgenis/molgenis-emx2
# feat(emx2): JSON api for reports

Fixes: [#4360](https://github.com/molgenis/molgenis-emx2/issues/4360)

What are the main changes you did:
- Added a JSON api mapping for reports
- Added a json download button to the reports view

how to test:
- Go to the pet store schema
- Click on reports, and select the first one
- Click on the download as json button
- See the json

todo:
- [x] support special case where the query returns 1 json objects
- [x] updated docs to explain us of this json
- [ ] add test for the single json blob
